<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkGenerationWindowClass</class>
 <widget class="QWidget" name="NetworkGenerationWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Generation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Insert the data to be used in 
new network generation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input neurons number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="inputs">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output neurons number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="outputs">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hidden layers number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hidden">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Neurons per a hidden layer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="per_hidden">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="connection">
     <item>
      <property name="text">
       <string>Connect to each neuron from previous layer including bias neuron</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Connect to each neuron from previous layer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No connections (Not recommended)</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Default weight function:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="default_weight">
       <item>
        <property name="text">
         <string>Random (-1.f, 1.f) (recommended)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random (0.f, 1.f)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Random (-1.f, 0.f)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.f</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5f</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.f (Not recommended)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-0.5f</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-1.f</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="eta_layout" stretch="0,1">
     <item>
      <widget class="QLabel" name="eta_label">
       <property name="text">
        <string>eta:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="eta">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.150000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="alpha_layout" stretch="0,1">
     <item>
      <widget class="QLabel" name="alpha_label">
       <property name="text">
        <string>alpha:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="alpha">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="moreButton">
     <property name="text">
      <string>More</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="generateButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
